packages/core/src/sass/currency-select-radio.scss

```diff
@@ -24,7 +23,7 @@
         height: 128px;
         justify-content: space-around;
         margin: 1.6rem 2rem 0;
-        width: 128px;
+        width: 20%;
```

Comment:
Why is `width` a `percentage` unit when height is in `px`?

Reply:
Will change one to be same as the other.